frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -34,16 +32,39 @@ public Car getCar(@PathVariable("carId") int carId) {
     @GetMapping(path = "/cars/{list}")
     public List<Car> getBatchCars(@PathVariable List<String> list) {
         List<Car> newList = new ArrayList<>();
-        for(String entry : list) {
-         newList.add(map.get(entry));
+        for (String entry : list) {
+            newList.add(map.get(entry));
         }
         return newList;
     }
+
     @PostMapping(path = "/cars", consumes = {"application/json"})
     public Map<String, Car> createCarList(@RequestBody List<Car> carList) {
```

Comment:
So this shouldn't return a Map like this, personal opinion but I don't like Json maps with variable keys, you can't put schemas on them. Just return a list. Also the function name should change as this is a full overwrite of the entire collection.

Reply:
Also shouldn't it just append to the list? Overwriting seems odd.

Reply:
Pete said it needs to be map. Alot of the other functions are also using this this map. If decided to change, perhaps i can refactor later on .